pages/news/we-count.vue

```diff
@@ -11,6 +11,7 @@
 
 <script>
 import axios from "axios"
+import Config from "@/assets/config"
```

Comment:
Why is there an `@` sign here?

```suggestion
import Config from "~/assets/config"
```